app/src/main/java/com/example/playlistmaker/PlayerActivity.kt

```diff
@@ -10,69 +16,168 @@ import androidx.activity.enableEdgeToEdge
+                    timer.text = SimpleDateFormat(
+                        "mm:ss",
+                        Locale.getDefault()
+                    ).format(STARTING_TIME_TRACK_CLOCK)
+                }
+            }
 
-            Glide.with(iconTrack)
-                .load(track.artworkUrl100.replaceAfterLast('/', "512x512bb.jpg"))
-                .placeholder(R.drawable.placeholder)
-                .fitCenter()
-                .transform(
-                    RoundedCorners(
-                        TypedValue.applyDimension(
-                            TypedValue.COMPLEX_UNIT_DIP,
-                            cornerImageTrack,
-                            resources.displayMetrics
-                        ).toInt()
-                    )
-                )
-                .into(iconTrack)
+    override fun onPause() {
+        super.onPause()
+        pausePlayer()
+    }
+    override fun onDestroy() {
+        mainThread.removeCallbacks(playerRunnable)
+        mediaPlayer.release()
+        super.onDestroy()
+    }
+    private fun playbackControl() {
+        when(playerState) {
+            STATE_PLAYING -> {
+                pausePlayer()
+            }
+            STATE_PREPARED, STATE_PAUSED -> {
+                startPlayer()
+            }
         }
+    }
+    private fun startPlayer() {
+        mediaPlayer.start()
+        play.setImageResource(R.drawable.pause)
+        playerState = STATE_PLAYING
+        trackClock()
+    }
+    private fun pausePlayer() {
+        mediaPlayer.pause()
+        mainThread.removeCallbacks(playerRunnable)
+        play.setImageResource(R.drawable.playbtn)
+        playerState = STATE_PAUSED
+    }
+    private fun preparePlayer() {
+        mediaPlayer.setDataSource(url)
+        mediaPlayer.prepareAsync()
+        mediaPlayer.setOnPreparedListener {
+            play.isEnabled = true
+            playerState = STATE_PREPARED
+        }
+        mediaPlayer.setOnCompletionListener {
+            play.setImageResource(R.drawable.playbtn)
+            timer.text = SimpleDateFormat(
+                "mm:ss",
+                Locale.getDefault()
+            ).format(STARTING_TIME_TRACK_CLOCK)
+            playerState = STATE_PREPARED
+            mainThread.removeCallbacks(playerRunnable)
+        }
+    }
+    private fun inflatePlayer(track: Track) {
+        val trackName: TextView = findViewById(R.id.trackName)
+        val artistName: TextView = findViewById(R.id.artistName)
+        val trackTime: TextView = findViewById(R.id.trackTime)
+        val iconTrack: ImageView = findViewById(R.id.iconTrack)
+        val trackClock: TextView = findViewById(R.id.trackClock)
+        val collectionName: TextView = findViewById(R.id.collectionName)
+        val releaseDate: TextView = findViewById(R.id.releaseDate)
+        val primaryGenreName: TextView = findViewById(R.id.primaryGenreName)
+        val country: TextView = findViewById(R.id.countryTrack)
+        val cornerImageTrack = 2f
```

Comment:
```suggestion
        val cornerImageTrack = 8f
```
По макетам радиус закругления - 8